components/DataTable/DataTable.tsx

```diff
@@ -103,6 +105,18 @@ export function DataTable<TData, TValue>({
     setRowSelection({});
   };
 
+  const exportHandler = handleExportSelected
+    ? () => {
+        const selectedData = table
+          .getSelectedRowModel()
+          .rows.map((r) => r.original);
+
+        handleExportSelected(selectedData);
+
+        setRowSelection({});
+      }
+    : null;
```

Comment:
Does this need to be memoised with `useCallback`?

Reply:
Done ✅